src/plenigo/internal/services/Service.php

```diff
@@ -147,7 +183,8 @@ protected function checkForErrors($response)
 
         $statusCode = $this->request->getStatusCode();
 
-        if ($statusCode != 200) {
+        // All 200 codes are good answers
+        if ($statusCode < 200 || $statusCode >= 300) {
```

Comment:
This if is for NO GOOD answers, so anything less than 200 and more or equal 300 is a BAD answer